lib/sign.js

```diff
@@ -57,11 +57,11 @@ function sign(options, config) {
       }
     });
     if (missingOptions.length) {
-      console.error();
+      console.error(); // eslint-disable-line no-console
```

Comment:
Only reason I can think of was to add some extra leading/trailing whitespace in the console.
I was trying to figure out how to wrap this in a single `console.error()` call, but it was a confusing mix of:

```
var errors = missingOptions.map(function(flag) {
  return `  error: missing required option '${flag}'`;
});
console.error(`\n${errors.join("\n")}\n`); // eslint-disable-line no-console
```

And I wasn't convinced that my theory would be any cleaner or clearer.